collect_app/src/main/res/layout/form_entry_end.xml

```diff
@@ -11,6 +11,8 @@ the specific language governing permissions and limitations under the License.
 <androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
Usually, we don't mix `android:` with `xmlns:` and have `xmlns:` first then `android:` and `app:` at the end.